skbio/alignment/_tabular_msa.py

```diff
@@ -37,8 +37,8 @@ class TabularMSA(SkbioObject):
     metadata : dict, optional
         Arbitrary metadata which applies to the entire MSA. A shallow copy of
         the ``dict`` will be made.
-    key : callable or metadata key, optional
-        If provided, defines a unique, hashable key for each sequence in
+    minter : callable or metadata key, optional
+        If provided, defines a unique, hashable key minter for each sequence in
```

Comment:
This wording here sounds like `minter` needs to be unique and hashable. Can you reword to indicate the key minter provides a unique, hashable key for each sequence?